falcon_multiqc/commands/query.py

```diff
@@ -248,24 +278,39 @@ def cli(
     """Query the falcon qc database by specifying what you would like to select on by using the --select option, and
     what to filter on (--tool_metric, --batch, or --cohort)."""
 
-    if multiqc and "sample" not in select:
-        click.echo(
-            "When multiqc report is selected, please ensure to select for sample.")
+    if (multiqc or csv) and not output:
+        click.echo("When using multiqc or csv option, please specify a directory to save in using the -o option")
         sys.exit(1)
 
     # Sqlaclehmy query that will be constructed based on this command's options.
     falcon_query = None
 
     ### ================================= SELECT  ==========================================####
+    # item in index 0 of select don't need to be added to join['joins']
+    # both select and filter options influence whether certain tables need to be joined, the following handles this
+
+    select = list(select) # needed for join section
```

Comment:
isn't select already a list?

Reply:
actually it's an array, but why does it need to be a list/

Reply:
Because I need to know the first item of it, unless there's a way of check what that is? - I use the list feature at line 71

Reply:
I'd rather not make it a list lol

Reply:
you can use array[0] too, no need to convert to list

Reply:
oh sweet haha

Reply:
Actually I think I do need it, because I can't find a way of inserting new items into the array (needed in line 295), it doesn't seem to have many methods

Reply:
Uhhh actually sorry in python there are no arrays, this is a list. My first comment stands, no need to say list(x), it's already one. Is it not?

Reply:
naa it's a tuple object, but not a regular python tuple it seems, it's a weird tuple with barely any methods 
![image](https://user-images.githubusercontent.com/71308285/97935435-a5f89e00-1dcc-11eb-9e7d-59a1a7d528aa.png)